docs/source/ephys_at_swc/getting_started.md

```diff
@@ -0,0 +1,133 @@
+# Getting Started
+
+Getting started with extracellular electrophysiology can be intimidating!
+With numerous acquisition setups, software packages, preprocessing, postprocessing,
+and analysis steps to consider, it's understandable to feel daunted.
+
+A good first step in getting started is to ask for advice. Every setup
```

Comment:
This is nice, but it sounds sightly patronising to me. There are people in SWC with 10x the NIU ephys experience combined.

Reply:
Good point, I was imaging it as a first-term PhD student whose just getting started. At the moment this page kind of mixes 'Getting Started' and 'Resources'. Maybe it would be better to split this, most of this page is 'Resources'. Then 'Getting Started' is just a paragraph aimed at new starters which points to Community, Examples and Resources.

Reply:
I would maybe change this page to "resources" and under a subheading of "Getting started", just introduce the different sections.

Reply:
In the end I split getting started into a completely different page will be good to see what you think, I'll re-request review once I've added in everyones pipelines.